Improve descriptions tag (#2772)
near/docs: docs/protocol/network/networks.md

id: networks
title: NEAR Networks
sidebar_label: Networks

> description: "Explore NEAR networks, from production-ready mainnet to testnet and localnet for development and testing."

Reviewer: ```suggestion
description: "Explore the different networks for development and testing."
```
